rsfbclient-core/src/connection.rs

```diff
@@ -145,10 +148,11 @@ pub trait FirebirdClientDbEvents: FirebirdClientDbOps {
     ) -> Result<(), FbError>;
 }
 
-#[derive(Debug, Eq, PartialEq, Copy, Clone)]
+#[derive(Debug, Eq, PartialEq, Copy, Clone, Default)]
 #[repr(u8)]
 /// Firebird sql dialect
 pub enum Dialect {
+    #[default]
     D1 = 1,
     D2 = 2,
     D3 = 3,
```

Comment:
I think dialect 3 should be the default one, as it is the default for the connection

```suggestion
#[derive(Debug, Eq, PartialEq, Copy, Clone, Default)]
#[repr(u8)]
/// Firebird sql dialect
pub enum Dialect {
    D1 = 1,
    D2 = 2,
    #[default]
    D3 = 3,
```